content/files/Akeneo PIM API.postman_collection.json

```diff
@@ -7,7 +7,7 @@
 	},
 	"item": [
 		{
-			"name": "Asset (2.1 only)",
+			"name": "Asset (2.1/2.2 only)",
```

Comment:
You will have to add each patch version in the future ? Isn't there another way to say that it's only for version 2.1 and further ?